router/network.go

```diff
@@ -134,10 +134,10 @@ func (network *NetHandleT) SendPost(ctx context.Context, structData integrations
 
 		req, err := http.NewRequestWithContext(ctx, requestMethod, postInfo.URL, payload)
 		if err != nil {
-			network.logger.Error(fmt.Sprintf(`400 Unable to construct "%s" request for URL : "%s"`, requestMethod, postInfo.URL))
+			network.logger.Error(fmt.Sprintf(`400 Unable to construct %q request for URL : %q`, requestMethod, postInfo.URL))
```

Comment:
[super-minor] backticks no longer needed
```suggestion
			network.logger.Error(fmt.Sprintf("400 Unable to construct %q request for URL : %q", requestMethod, postInfo.URL))
```